addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -1067,6 +1078,13 @@ var SnippetsMenu = Widget.extend({
                 return;
             }
             this._activateSnippet($target);
+            if (this.options.enableTranslation) {
+                // Ensure translate mode has the toolbar.
+                this._updateLeftPanelContent({
```

Comment:
Good catch! I'll look into fixing it.
Now we _could_ just return in `start`, not do the binding so we don't update on click, and add the toolbar in `start` so we just _always_ have it, but I don't know if that would really be correct. I'll give it a shot but my concern is that it would cause strange behavior when we are not in content that can be altered with the toolbar.

Reply:
Update: Tried it, it works fine.